specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/virtualNetwork.json

```diff
@@ -823,6 +824,48 @@
     }
   },
   "definitions": {
+    "ServiceAssociationLinkFormat": {
+      "properties": {
+        "linkedResourceType": {
+          "type": "string",
+          "description": "Resource type of the linked resource."
+        },
+        "link": {
+          "type": "string",
+          "description": "Link to the external resource."
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Provisioning state of the ServiceAssociationLink resource."
+        }
+      },
+      "description": "Properties of ServiceAssociationLink."
+    },
+    "ServiceAssociationLink": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ServiceAssociationLinkFormat",
+          "description": "Resource navigation link properties format."
+        },
+        "name": {
```

Comment:
Please check if this is readonly

Reply:
The name of the service association link should not be marked as read only